README.md

```diff
@@ -294,6 +294,9 @@ settings. Note that `~/.spring.rb` is loaded *before* bundler, but
 projects without having to be added to the project's Gemfile, require
 them in your `~/.spring.rb`.
 
+`config/spring_preboot.rb` is also loaded before bundler and before a 
+server process is started, it can be used to add new top-level commands.
```

Comment:
Sorry for the late reply, but I think this might be better named `config/spring_client.rb` - that would make it clearer in my mind that there is a "client" and a "server" side to spring, and that this file allows modifying the client side (whereas the other config files operate on the server side).